awscrt/io.py

```diff
@@ -301,7 +373,6 @@ def create_client_with_mtls_pkcs12(pkcs12_filepath, pkcs12_password):
         opt = TlsContextOptions()
         opt.pkcs12_filepath = pkcs12_filepath
         opt.pkcs12_password = pkcs12_password
-        opt.verify_peer = True
```

Comment:
What motivated this?

Reply:
The default `TlsContextOptions()` constructor already sets this true
it was weird that this one factory function was setting it explicitly when the other factory functions weren't